src/lib/swapper/LifiSwapper/LifiSwapper.ts

```diff
@@ -129,7 +130,12 @@ export class LifiSwapper implements Swapper<EvmChainId> {
   /**
    * Get transactions related to a trade
    */
-  async getTradeTxs(_tradeResult: TradeResult): Promise<TradeTxs> {
-    return await Promise.reject(new SwapError('LifiSwapper: getTradeTxs unimplemented'))
+  async getTradeTxs(tradeResult: TradeResult): Promise<TradeTxs> {
+    // the tradeId is currently a lifi route ID
+    // TODO: determine a way to return actual txId per step (there could be mulitple)
```

Comment:
Very nice. Product-wise, our current heuristics for this is to display the outbound Tx only, but aggregating the multiple Txids of a single treade would be great, especially trades failing mid-way (which definitely happens with flows like the li.fi one, as you've found out!).

Definitely not in the scope of this PR or this milestone altogether, but would be great to have a `li.fi` `unchained-client` parser for that